pkgs/os-specific/darwin/trash/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, frameworks, perl } :
+stdenv.mkDerivation rec {
+  version = "0.9.0";
+  name = "trash-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ali-rantakari";
+    repo = "trash";
+    rev = "f68ad25a02e24cc58eb8ef9a493d6dc0122bcd8f";
```

Comment:
It would be better to use a release-tag instead of the revision. As there is no release-tag for 0.9.0 could you please ask the Author to create one?